tests/plugins/snapshot_management/snapshot_management.yaml

```diff
@@ -0,0 +1,131 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
```

Comment:
This doesn't run unless you add it explicitly into CI.

Reply:
We should be able to detect this issue automatically and fail CI when some tests are never run. I totally didn't know about explicitly registering a test folder until now. I always assumed that if a docker file is not detected, it will look for the closest docker file in the parent folders.

Reply:
@dblock aah, sorry I did it again
Last time I was doing observability api and I saw 'plugin' in URI ,I also put It in the different directory
Getting confused all the time
I'll try to finish it by the end of the week

Reply:
I added #705 yesterday :)